src/NCPkgFilterRPMGroups.cc

```diff
@@ -244,7 +244,6 @@ bool NCPkgFilterRPMGroups::showRPMGroupPackages ( const std::string & label, YSt
 
     // get the package list and sort it
     std::list<ZyppSel> pkgList( zyppPkgBegin (), zyppPkgEnd () );
-    pkgList.sort( sortByName );
```

Comment:
I guess (!) that they are also unnecessary, but we'd have to spend some time to make sure, in particular to test this thoroughly.

That widget appears to do its own sorting anyway, regardless of any previous sorting. In the case of those RPM groups that sorting hurt pretty much, since it would always sort the complete zypp pool before even beginning to filter out the relevant ones, i.e. it sorted the items just to throw most of the neatly sorted list away (because the filter did not apply).